man/SAPHanaSR.py.7

```diff
@@ -269,6 +309,7 @@ Please report any other feedback and suggestions to feedback@suse.com.
 \fBSAPHanaSR-monitor\fP(8) , \fBSAPHanaSR-showAttr\fP(8) ,
 \fBSAPHanaSR-manageProvider\fP(8) , \fBSAPHanaSR-hookHelper\fP(8) ,
 \fBcrm_attribute\fP(8) , \fBsudo\fP(8) , \fBsudoers\fP(5), \fBpython\fP(8) ,
+\fBitables\fP(8) , \fBss\fP(8) ,
```

Comment:
Typo? iptables instead of itables?

Reply:
thanks!
fixed - also in my local angi branch